### DISCUSSION TITLE
Are there any known limitations in the number of lists items in Revealjs presentation?

### DISCUSSION THREAD

**Discussion by friosavila** [quarto-dev/quarto-cli#10388] [Q&A]
### Description

Has there been any reports that a presentation becomes unstable when too many lists (numbered or unnumbered) are created in a revealjs presentation?
Im currently having that issue, and the only way to correct it so far is to "drop" some of those lists.

**Comment by mcanouil** [comment]
What does "unstable" mean? How many is "too many lists"?

Would you mind sharing some concrete/reproducible example?

**Reply by friosavila** [reply]
Im attaching the document. It uses the format clean-revealjs
[CC_ressp.txt](https://github.com/user-attachments/files/16392777/CC_ressp.txt)

**Reply by friosavila** [reply]
this version "crasshes" after the Markdown formating example.
also currently using Quarto 1.4.533

**Reply by mcanouil** [reply]
Is it "unstable" with the default format?

**Reply by friosavila** [reply]
Yes.  Neither the outline (stops at YAML header) not the flow of the slides works
(currently if I get to just before "othertools" it returns to the first slide),

**Reply by mcanouil** [reply]
I don't think the issue is the list numbering, but more on the fact that you are using undeclared raw HTML instead of using markdown fenced div or raw code block.

In particular the issue is your big tabset slide which mixing things.
I suggest you fix the syntax of your buggy tabset slide using proper markdown, properly escaping what needs to be escaped, ensuring every fenced divs are properly closed, etc.
